src/IO/H5/Python/IterElements.py

```diff
@@ -49,6 +49,34 @@ def dim(self):
     def logical_coordinates(self):
         return logical_coordinates(self.mesh)
 
+    @cached_property
```

Comment:
I don't understand why this change is in this commit

Reply:
move to the correct commit